clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -23,7 +23,11 @@
 import org.apache.kafka.clients.ClientUtils;
 import org.apache.kafka.clients.KafkaClient;
 import org.apache.kafka.clients.NetworkClient;
+<<<<<<< HEAD
+import org.apache.kafka.clients.CommonClientConfigs;
+=======
 import org.apache.kafka.clients.StaleMetadataException;
+>>>>>>> trunk
```

Comment:
I think these are left from your merge. You need to fix and pick which line during the merge. This will break the compilation

Reply:
Fixed, thanks. Don't know how this slipped through as merge didnt show me that file so thought my merge was complete.